integration_test/third_party_apps_data/applications/postgresql/centos_rhel/install

```diff
@@ -22,3 +22,8 @@ sudo /usr/pgsql-14/bin/postgresql-14-setup initdb
 sudo service postgresql-14 restart
 
 sudo su postgres -c "psql postgres -c \"ALTER ROLE postgres WITH PASSWORD 'abc123';\""
+sudo su postgres -c "psql postgres -c \"CREATE TABLE test1 (id serial PRIMARY KEY);\""
```

Comment:
The `sudo su postgres` lines seem not related to the installation. Can we move them to:

https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/third_party_apps_data/applications/postgres/exercise

Similar to https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/third_party_apps_data/applications/nginx/exercise, which is where commands go when they are not distro specific and the goal is to trigger some metrics / logs for testing instead of installing the application.

Reply:
Sure can do that, would the modification of the postgres user still be included in the `install` file then or moved to `exercise` from your perspective?